src/ape/api/address.py

```diff
@@ -54,6 +54,18 @@ def address(self) -> AddressType:
             :class:`~ape.types.AddressType`
         """
 
+    def __eq__(self, __o: object) -> bool:
+        """
+        Compares AddressAPI/str objs
+
+        Returns:
+            bool: comparison result
+        """
+
+        if isinstance(__o, str):
+            return self.address == __o
+        return super.__eq__(self, __o)
```

Comment:
```suggestion
        return super().__eq__(self, __o)
```

I thought we had to call `super`, no?

Reply:
I'll put it back to a more conventional implementation but these behave the same (I think/doesn't break, lol):
```python
super().__eq__(__o)

super.__eq__(self, __o)
```

Reply:
^ the syntax highlighting is suspicious

Reply:
question, why not just `return convert(self, AddressType) == convert(__o, AddressType)`?

Reply:
also, `other` would be better than `__o`

Reply:
Much cleaner, I originally used the `__o` from vscode code stub auto-generation for the override.
03075bf